inc/assets/css/modern-admin-style.css

```diff
@@ -0,0 +1,21 @@
+.p-1 {
+    padding: 0.25rem !important;
```

Comment:
Can we try removing !important

Reply:
Its same value - https://share.getcloudapp.com/wbuvY4Ex, but there was one case (not remember exactly now) but a external entity was overriding this CSS, that's why added important over here.